[2025-02-18T04:30:54.378+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:30:54.606+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-02-23T00:00:00+00:00 [queued]>
[2025-02-18T04:30:54.645+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-02-23T00:00:00+00:00 [queued]>
[2025-02-18T04:30:54.649+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:30:54.823+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-02-23 00:00:00+00:00
[2025-02-18T04:30:54.949+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=251) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:30:54.953+0000] {standard_task_runner.py:72} INFO - Started process 267 to run task
[2025-02-18T04:30:54.946+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-02-23T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpp7dxwdb2']
[2025-02-18T04:30:54.962+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask hello_task
[2025-02-18T04:30:55.028+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:30:55.212+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-02-23T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:30:55.215+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:30:55.452+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-23T00:00:00+00:00'
[2025-02-18T04:30:55.458+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:30:55.513+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:30:55.517+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:30:55.560+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:30:55.568+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-02-23T00:00:00+00:00, execution_date=20240223T000000, start_date=20250218T043054, end_date=20250218T043055
[2025-02-18T04:30:56.046+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:30:56.129+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-18T16:33:57.908+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T16:33:57.967+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-02-23T00:00:00+00:00 [queued]>
[2025-02-18T16:33:57.994+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-02-23T00:00:00+00:00 [queued]>
[2025-02-18T16:33:57.997+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T16:33:58.037+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-02-23 00:00:00+00:00
[2025-02-18T16:33:58.059+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1533) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T16:33:58.062+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-02-23T00:00:00+00:00', '--job-id', '375', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmp1absg4pv']
[2025-02-18T16:33:58.069+0000] {standard_task_runner.py:105} INFO - Job 375: Subtask hello_task
[2025-02-18T16:33:58.068+0000] {standard_task_runner.py:72} INFO - Started process 1539 to run task
[2025-02-18T16:33:58.174+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:33:58.456+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-02-23T00:00:00+00:00 [running]> on host 8720d9c228eb
[2025-02-18T16:33:58.463+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:33:59.480+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-23T00:00:00+00:00'
[2025-02-18T16:33:59.500+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T16:33:59.575+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T16:33:59.587+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T16:33:59.664+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T16:33:59.669+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-02-23T00:00:00+00:00, execution_date=20240223T000000, start_date=20250218T163357, end_date=20250218T163359
[2025-02-18T16:34:00.018+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T16:34:00.349+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T16:34:00.378+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
